File: 参考/comp9041-454/9041-labonserver/test11/devowel_file.py
#!/usr/bin/python3
import sys,re
name = sys.argv[1]
f = open(name,'r')
data = f.readlines()
f.close()
f = open(name,'w')
for each in data:
    each = re.sub('[AEIOUaeiou]','',each)
    print(each,end='',file=f)
f.close()

File: 参考/comp9041-454/9041-labonserver/test11/devowel_file.pl
#!/usr/bin/perl -w
@need_print = ();
$name = $ARGV[0];
while ($line = <>){
    $line =~ s/[aeiou]//ig;
    push @need_print, $line;
}
open F,'>',"$name" or die;
print F @need_print;
close F;
